src/main/java/com/backbase/ct/bbfuel/data/ContentServicesDataGenerator.java

```diff
@@ -0,0 +1,27 @@
+package com.backbase.ct.bbfuel.data;
+
+import com.backbase.dbs.contentservices.client.v2.model.AntivirusScanTrigger;
+import com.backbase.dbs.contentservices.client.v2.model.Repository;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ContentServicesDataGenerator {
+
+    private ContentServicesDataGenerator() {
+    }
+    
+    public static List<Repository> createRepositories(String... repositoryIds) {
+        return Arrays.stream(repositoryIds)
+            .map(repositoryId ->
+                new Repository()
+                    .repositoryId(repositoryId)
+                    .name(repositoryId)
+                    .description(String.format("Repository for %s", repositoryId))
+                    .implementation("DB")
+                    .versioningEnabled(false)
+                    .isPrivate(false)
```

Comment:
Is it possible to use a private repository? The content only needs to be accessed by payment service.

Reply:
As discussed, since accessing private repository from client API is not possible, going with public repository. In documentation, I will add a link to content service docs to provide more info on private/public repositories.